int32    task_ID
int32    known_power
float32  current_distance
bool     istarget
bool     iscomplete
bool     isexplored
bool     isallocated

int32    task_power
bool     isupdate
